LICENSE.txt

```diff
@@ -652,7 +652,7 @@ Also add information on how to contact you by electronic and paper mail.
   If the program does terminal interaction, make it output a short
 notice like this when it starts in an interactive mode:
 
-    {{project}}  Copyright (C) {{year}}  {{fullname}}
+    Copyright (C) <year> <name of author>
```

Comment:
@DiegoPino I think https://www.gnu.org/licenses/gpl-3.0.txt is the standard we're using, not the other